sources/lib/DependencyInjection/Configuration.php

```diff
@@ -42,9 +42,34 @@ public function getConfigTreeBuilder()
                     ->prototype('array')
                         ->children()
                             ->scalarNode('dsn')->isRequired()->end()
-                            ->scalarNode('class:session_builder')->defaultValue('\PommProject\ModelManager\SessionBuilder')->end()
+                            ->scalarNode('class:session_builder')
+                                ->defaultNull()
+                                ->beforeNormalization()
+                                    ->always()
+                                    ->then(function ($v) {
+                                        @trigger_error('class:session_builder is deprecated since 2.3 and is going to be removed in 3.0 Use service and service id instead.', E_USER_DEPRECATED);
```

Comment:
@stof Why can't I see this deprecation notice in a Profile/Logs/Deprecations? Where should I look for it.